joomla/joomla-cms
# [4.3][bug] Fix the infobar gap

Pull Request for Issue #39792 .

### Summary of Changes

Fixes the 4.2 up merge by removing a redundant line


### Testing Instructions

- Check that the infobar doesn't have any gap
- Check that an empty folder has correctly entered the upload icon

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Link to documentations
Please select:
